src/App.tsx

```diff
@@ -37,6 +37,104 @@ function App() {
   return (
     <Router>
       <AppHeader loggedIn={loggedIn} setUserId={setUserId} />
+      <link rel="preconnect" href="https://fonts.gstatic.com" />
```

Comment:
Use [this](https://stackoverflow.com/questions/40769551/how-to-use-google-fonts-in-react-js) over that you have.